subprojects/core/src/test/groovy/org/gradle/internal/fingerprint/impl/DefaultFileCollectionSnapshotterTest.groovy

```diff
@@ -215,8 +215,7 @@ class DefaultFileCollectionSnapshotterTest extends Specification {
 
     void assertEmptyTree(FileCollection fileCollection) {
         def snapshots = snapshotter.snapshot((FileCollectionInternal) fileCollection)
-        assert snapshots.size() == 1
-        assert snapshots[0] == FileSystemSnapshot.EMPTY
```

Comment:
Do we still need `FileSystemSnapshot.EMPTY`?

Reply:
Yes.